xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1044 @@
+        target_project: OBS:Server:Unstable:CI
+  filters:
+    event: pull_request
+rebuild_master:
+  steps:
+    - rebuild_package
+        project: home:Admin
+        package: ctris
+  filters:
+    event: push
+    branches:
+      only:
+        - master</screen></para>
+
+      <para>The file can contain multiple workflows, but each one needs an
+      <emphasis>unique</emphasis> name. The previous example contains two
+      workflows: <emphasis>main_workflow</emphasis> and
+      <emphasis>rebuild_master</emphasis>.</para>
+
+      <para>OBS assumes the configuration file is placed in your target branch
+      and not in other branches or forks.</para>
+
+      <note>
+        <title>Workflows Cleanup</title>
+
+        <para>Workflows are cleaned up after themselves when pull requests are
+        closed or merged. Projects in which you branched or linked packages
+        will be deleted, the same thing for the repositories you configured
+        for those projects.</para>
+
+        <para>If you decide to reopen the pull request, this will also be
+        handled by workflows. Your projects, their branched/linked packages
+        and their configured repositories will be restored.</para>
+      </note>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.steps">
+        <title>Steps</title>
+
+        <para>Currently, the supported steps are:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Branch a Package (branch_package).</para>
+          </listitem>
+
+          <listitem>
+            <para>Link a Package to a Project (link_package).</para>
+          </listitem>
+
+          <listitem>
+            <para>Configure Repositories/Architectures for a Project
+            (configure_repositories)</para>
+          </listitem>
+
+          <listitem>
+            <para>Rebuild a Package (rebuild_package)</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>Some of these steps need to get the sources from the SCM pull
+        request. Read <link
+        linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.steps.service">OBS
+        SCM Service</link> for more details.</para>
+
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.branch_a_package">
+          <title>Branch a Package</title>
+
+          <para>Given we have a source package called<emphasis>
+          obs-server</emphasis> coming from a source project called
+          <emphasis>OSB:Server:Unstable</emphasis>, and a target project
+          called <emphasis>OBS:Server:Unstable:CI</emphasis>, this step will
+          branch that package onto the target project, keeping in mind
+          that:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>For a pull request event it will go to
+              <emphasis>OBS:Server:Unstable:CI:$MY_SCM_ORG:$MY_SCM_PROJECT:PR-$MY_PR_NUMBER/obs-server</emphasis>.</para>
```

Comment:
I think we should use some real world example. Expressing things in variable names like `$MY_SCM_ORG` might work for OBS developers but for sure not for our customers...